### PR TITLE
i18n(es): update `sidebar`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- Closes # N/A
- What does this PR change? Give us a brief description.
Updates `sidebar` based on this commit eeba06ea7df962e8f0520e145d28b8c17cd32c18

### CODE REVIEW

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 1]
```diff
@@ -93,15 +147,15 @@ starlight({
...
 				// Un grupo anidado de enlaces para constelaciones estacionales.
 				{
 					label: 'Estacional',
 					items: [
-						{ label: 'Andrómeda', link: '/constellations/andromeda/' },
-						{ label: 'Orión', link: '/constellations/orion/' },
-						{ label: 'Osa Menor', link: '/constellations/ursa-minor/' },
+						'constellations/andrómeda',
+						'constellations/orión',
+						'constellations/ursa-menor',
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/sidebar.mdx:158]:
```suggestion
						'constellations/andromeda',
						'constellations/orion',
						'constellations/ursa-minor',						
```

The existing translations did not translate slugs but only titles so we should keep it consistent (or change that everywhere on the page if you prefer, your call).

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 2]
```diff
@@ -310,8 +363,7 @@ starlight({
 			items: [
 				// Un enlace con una insignia "Stub" amarilla.
 				{
-					label: 'Sirio',
-					link: '/stars/sirius/',
+					slug: 'stars/sirius',
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/sidebar.mdx:366]: The highlight of this code block should be updated from `10` to `9` to match this change.

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 3]
```diff
@@ -429,6 +481,43 @@ Navegar por la documentación en portugués de Brasil generará la siguiente bar
...
+			items: [
+				{ label: 'Andrômeda', link: '' },
+				{ label: 'Escorpião', link: '' },
+			],
+		},
+	]}
+/>
+En sitios multilingües, el valor de `slug` no incluye la parte del idioma de la URL.
+Por ejemplo, si tienes páginas en `en/intro` y `pt-br/intro`, el slug es `intro` al configurar la barra lateral.
+
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/sidebar.mdx:520]: The first code block of the "Collapsing groups" section is missing a [change](https://github.com/withastro/starlight/pull/1874/files#diff-f2469f170cfdf3bc0930d7a06705fe128dd16032760fff8618641d1dc5d5edadL444-L447) but I cannot suggest a change for it:


```diff
-			items: [
-				{ label: 'Andrómeda', link: '/constellations/andromeda/' },
-				{ label: 'Orion', link: '/constellations/orion/' },
-			],
+			items: ['constellations/andromeda', 'constellations/orion'],
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few comments/suggestions but overall the translation is looking really great :thumbsup:

**Comment by dreyfus92** [comment]
Thank you for the review @HiDeoo 😁

**Review by HiDeoo — APPROVED** [review]
Thanks for the update, this is looking great :raised_hands: (I removed duplicated `items` from the last section as it was a very minor edit, hope you don't mind)

**Pull request merged**